about.md

```diff
@@ -0,0 +1,145 @@
+
+Let's say for instance that you have a page that you'd like to back-date to last January.  Maybe that was the date you started your project or organization and you want the page to reflect that.  All you do is... type in that date!  There's no hoops to jump through, you don't have to go into some management console to modify your post, you just change the date.  You also don't have to worry that that minor edit you made to an older file will silently change a page's date to today's, thoroughly confusing all your visitors.
+
+* Add YAML Example block?
+
+## Flexibility and Customization
+
+Pico is not just easy to use, it's *powerful*!  Pico has two very large avenues of customization, [Themes and Plugins][Customization].
+
+### Themes...
+
+* Snappier section titles
+
+Pico's default theme is *not* intended for production use.  It is provided as a great, but minimal starting place for you to develop your own customized website.  If you aren't familiar with HTML, fear not, we have an ever-growing variety of community-created themes available [here on our site][SiteThemes], as well as some more on [our wiki][WikiThemes].
+
+Pico's "Themes" are built using [Twig Templates][Twig].  Twig is a template engine which provides an easy and powerful way to customize your website.  You can use small amounts of Twig to add dynamic content to a mostly-static HTML website, or use large amounts of Twig to build in some really incredible features.
+
+* link to cookbook? docs?
+
+And if that's not enough creative power for you, you can check out Pico's Plugin system.
+
+### Plugins...
+
+Pico's Plugin system allows for users to extend Pico's functionality by hooking in their own PHP code.  Along with Themes, we also have a growing library of community-developed plugins you can use to add new features to your Pico site.  You can find these plugins [here on our site]({{ site.github.url }}/customization/#plugins), and even more on [our wiki]({{ site.gh_project_url }}/wiki/Pico-Plugins).
+
+* **@PhrozenByte** any more to say about Plugins?
+
+## Getting Started
+
+Ready to try Pico for yourself?  Head on over to our [Download][Download] page to get yourself a copy.  You can find more information on building your own Pico site in our [Documentation][Docs].  And of course, **we're here to help**!  If you require any assistance, or if you find a bug in Pico, let us know!  Check out the [Getting Help][GettingHelp] section of our Documentation for more details.  We appreciate your feedback!  Pico is a community-driven project and we need *your* feedback to keep making it better!
+
+* **General Notes**
+* Overview of Why's
+* Expand YAML in main documentation.
+
+
+* **VS Other CMS's**
+* configuration
+* limitations
+
+* **Community**
+* Start a discussion about how people got started with Pico.
+  * LUP and maybe an interview to build community?
```

Comment:
Lol, I left that one vague in case you came across it.  I felt it warranted its own discussion, but again, it's going to be off-topic. :unamused:

[**L**inux **U**n**P**lugged](http://www.jupiterbroadcasting.com/show/linuxun/)

It's a podcast from [Jupiter Broadcasting](http://www.jupiterbroadcasting.com/).  There's a lot of free-form discussion in it.  During the show there's a live audio chat open to audience members (though it's typically the same contributes).

A little over a year ago, someone mentioned Pico during an off-hand discussion about alternative CMS's.  It was literally something that was passed over without much thought, but I liked the description enough to come check it out.

Now, that was *just* before your "in-hostile takeover" ( / revival) of the project.  It would have been when `0.8` was current, because that's the version I started on.  It's been so long, and so much work has been done that Pico feels entirely different now.

So, I thought that in the future* it would be interesting to get Pico featured on there or the [Linux Action Show](http://www.jupiterbroadcasting.com/show/linuxactionshow/) (their main show) to raise awareness of it.  It could be a news article for them to read or a full blown interview about the project.

(*maybe 1.1 or whenever the more blogging-oriented work is done.  I don't know, blogging just seems like it would be the "mainstream" expectation of a CMS.)

Anyway, what are your thoughts on this idea?